pipeline/vars/common.groovy

```diff
@@ -99,33 +87,42 @@ def runTestSuite() {
         rc = 1
         echo "Encountered an error"
     }
-
     // Forcing cleanup
     try {
         cleanup_cmd = "PYTHONUNBUFFERED=1 ${env.WORKSPACE}/.venv/bin/python"
         cleanup_cmd += " run.py --cleanup ${instanceName}"
         cleanup_cmd += " --osp-cred ${env.HOME}/osp-cred-ci-2.yaml"
         cleanup_cmd += " --log-level debug"
-
         sh(script: "${cleanup_cmd}")
     } catch(Exception ex) {
         echo "WARNING: Encountered an error during cleanup."
         echo "Please manually verify the test artifacts are removed."
     }
-
     if (rc != 0) {
         error("Test execution has failed.")
     }
-
+    return rc
 }
 
-def sendEMail(def subjectPrefix) {
+def sendEMail(def subjectPrefix,def test_results) {
     /*
         Send an email notification.
     */
+    def body = readFile(file: "pipeline/vars/emailable-report.html")
+    body += "<body><u><h3>Test Summary:</h3></u><br />"
+    body += "<p>Console logs are available at ${env.BUILD_URL}/console</p><br />"
+    body += "<table><tr><th>Test Stage Name</th><th>Status</th>"
+    for (test in test_results){
+    def res
+    if (test.value == 0){res = "PASS"} else {res = "FAIL"}
+    body += "<tr><td>${test.key}</td><td>${res}</td></tr>"
+    }
+    body +="</table> </body>"
+
     emailext(
+        mimeType: 'text/html',
         subject: "${subjectPrefix} test suite execution summary of ${env.composeId}",
-        body: "Console logs are available at ${env.BUILD_URL}/console",
+        body: "${body}",
         from: "cephci@redhat.com",
         to: "cephci@redhat.com"
```

Comment:
On success, can you send an email to `ceph-qe-list` and
On failure, can you send an email to `cephci`

We can pick this up in a separate PR if needed